managed/services/agents/mongodb.go

```diff
@@ -187,6 +187,11 @@ func v225Args(exporter *models.Agent, tdp *models.DelimiterPair, listenAddress s
 			enabled:     false,
 			enableParam: "--collector.topmetrics",
 		},
+		// disabled until we have better information on the resources usage impact
+		"shards": {
+			enabled:     false,
+			enableParam: "--collector.shards",
+		},
```

Comment:
we don't have it in old versions, so let's drop it

Reply:
Removed.